JS/Lesson34/goksel/recipe-app/src/modules/recipes/models/index.tsx

```diff
@@ -0,0 +1,73 @@
+export interface Recipe {
+    idMeal : string;
+    strMeal : string;
+    strDrinkAlternate?: string;
+    strCategory: Categories;
+    strArea : Areas;
+    strInstructions: string;
+    strMealThumb: string;
+    strTags: string;
+    strYoutube: string;
+    strIngredient1:string;
+    strIngredient2:string;
+    strIngredient3:string;
+    strIngredient4?:string;
+    strIngredient5?:string;
+    strMeasure1:string;
+    strMeasure2:string;
+    strMeasure3:string;
+    strMeasure4?:string;
+    strMeasure5?:string;
+    strSource:string;
+    strImageSource?:string;
+    strCreativeCommonsConfirmed?:string;
+    dateModified?:string;
+};
+
+export enum Categories {
```

Comment:
You have a Categories object defined in the NewRecipeForm folder, it would be better if you use this interface instead. Same for Areas

Reply:
I just removed Categories and Areas objects defined in the NewRecipeForm folder and ı used these interfaces. Should ı change or is this fine like that?